airflow/cli/commands/jobs_command.py

```diff
@@ -27,6 +27,7 @@ def check(args, session=None):
     """Checks if job(s) are still alive"""
     if args.allow_multiple and not args.limit > 1:
         raise SystemExit("To use option --allow-multiple, you must set the limit to a value greater than 1.")
+
```

Comment:
If we want to use `default` we need to remove this condition, it's not useful again, right?
```python
if args.hostname:
        query = query.filter(BaseJob.hostname == args.hostname)
```
and add a filter by hostname on the query below

Reply:
The `if args.hostname:` is indeed not necessary anymore. but the filter is working fine as it is now, or I am not sure to understand your point )

Reply:
Just a nit, we could have this:
```python
 query = (
        session.query(BaseJob)
        .filter(BaseJob.state == State.RUNNING)
        .filter(BaseJob.hostname == args.hostname)
        .order_by(BaseJob.latest_heartbeat.desc())
    )
```
WDYT?